### PR TITLE
Use existing SVG components

### PR DESCRIPTION
**Pull request by regiskuckaertz**
## Why are you doing this?

Instead of linearly embedding the content of SVGs into the new components, we use the existing machinery. This reduces the code complexity and potential for errors.

The PayPal logo is a PNG, but I'm not sure how to deal with these, so I put it inside `assets/images` but it's not appearing in the rendered page 🤔 I'm using an IMG for the credit card icon too (even though it's an SVG), but happy to change that if necessary.

TODO
- [x] Fix PayPal rendering
- [x] Maybe fix roundel and card images

[**Trello Card**](https://trello.com/c/xvQanbaA/786-npf-moves-svgs-into-react-components)

### CODE REVIEW

**Comment by regiskuckaertz** [comment]
@Ap0c all done 👍

File: assets/components/svgs/enveloppe.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import React from 'react';
+
+// An enveloppe icon
+export default function SvgEnveloppe() {
+
+  return (
+    <svg className="svg-enveloppe" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 10" preserveAspectRatio="xMinYMid">
```
Comment (Ap0c) [inline assets/components/svgs/enveloppe.jsx]: "envelope"?

File: assets/components/svgs/paypal.jsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// @flow
+
+import React from 'react';
+
+// The generic credit card logo used on checkouts.
```
Comment (Ap0c) [inline assets/components/svgs/paypal.jsx]: "credit card" -> "PayPal"

File: assets/components/svgs/newCreditCard.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import React from 'react';
+
+// The generic credit card logo used on checkouts.
+export default function SvgCreditCard() {
+
+  return (
+    <svg className="svg-credit-card" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 22 14" preserveAspectRatio="xMinYMid">
```
Comment (Ap0c) [inline assets/components/svgs/newCreditCard.jsx]: Possibly worth naming this component "svg-new-credit-card"? It shouldn't matter, but this name is already taken by the component in `creditCard.jsx`.

**Pull request merged**